docs/UserGuide.md

```diff
@@ -1,36 +1,38 @@
----
-layout: page
-title: User Guide
----
```

Comment:
Probably best to leave this part in for better viewing in the website of the user guide